package.json

```
@@ -105,7 +105,8 @@
     "next-release": "standard-version --scripts.prebump=./build/next-version.js --skip.commit=true --skip.tag=true",
     "sri-update": "grunt build && node build/sri-update && git add sri-history.json",
     "sri-validate": "node build/sri-update --validate",
-    "fmt": "prettier --write .*.json *.{json,md,js} **/*.ts './{.circleci,.github,build,doc,lib,locales,test,typings}/**/*.{json,md,js,ts,html}'"
+    "fmt": "prettier --write .*.json *.{json,md,js} **/*.ts './{.circleci,.github,build,doc,lib,locales,test,typings}/**/*.{json,md,js,ts,html}'",
+    "postinstall": "build/postinstall.sh"
```

We cannot use `postinstall` for this. The `postinstall` script happens on users' machines (post installation).

We could use `prepare` instead.

```suggestion
    "prepare": "build/postinstall.sh"
```

---

I thought prepare was only run on `publish` not install.

---

From https://docs.npmjs.com/cli/v8/using-npm/scripts, the `prepare` script is run:

- Runs any time before the package is packed, i.e. during npm publish and npm pack
- Runs BEFORE the package is packed
- Runs BEFORE the package is published
- Runs on local npm install without any arguments